app/controllers/jobseekers/job_applications/self_disclosure_controller.rb

```diff
@@ -0,0 +1,95 @@
+class Jobseekers::JobApplications::SelfDisclosureController < Jobseekers::BaseController
+  include Wicked::Wizard
+
+  before_action :redirect_when_self_disclosure_not_pending, except: :completed
+  before_action :form, except: :completed
```

Comment:
This feels a little counter-intuitive - the method uses ||= as though it is a normal method, but it doesn't need to. Maybe it should be called set_form, and then controller uses @form throughout as it can be depended on to be set